src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -9,7 +9,7 @@ define(function (require, exports, module) {
     var Commands           = brackets.getModule("command/Commands");
     var FileSystem         = brackets.getModule("filesystem/FileSystem");
     var HTMLRewriter       = brackets.getModule("filesystem/impls/filer/lib/HTMLRewriter");
-    var SidebarView        = brackets.getModule("project/SidebarView");
+    var SidebarView        = brackets.getModule("project/SidebarView");    
```

Comment:
undo this change

Reply:
Both lines are exact same, not sure what to undo.

Reply:
See the extra green at the end of the line?  That's whitespace that you've added without actually changing the content of the file.  People try to avoid this in code because it ends up obscuring the history of this file/line in git--it looks like you changed it, but actually, it wasn't really changed in this patch substantively, just a cosmetic thing.